docs/datasets/README.md

```diff
@@ -0,0 +1,43 @@
+# Generating an AIP database
+
+Disclaimer: to update the database one would need around 1TB of storage available and about 30h of computation time. 
```

Comment:
Nah not really, you can process the tar.gz (or was it just gz?) files one-by-one. The parser checks if it it's zipped and unzips it in memory. But to the collection of all zipped data was somewhere around 400GB if my memory serves.

Reply:
I made the disclaimer less precise, because the data might change in size in the future (as well as the processing time) ;)